### DISCUSSION TITLE
Allow the overivew dashboard to be hidden from "users"

### DISCUSSION THREAD

**Discussion by jgthornburgh** [home-assistant/frontend#11244] [Other feature requests]
in 107 the overview dashboard is able to be hidden by administrators when they make another dashboard default. However, users are unable to set another as default since they don't have access to customize. It would be good to have the option in the dashboards manager to set the overview to admins only, or for it to not show if another dashboard has been created.

**Comment by wojciechczyz** [comment]
Did you found any workaround for this? Thank you

**Comment by jgthornburgh** [comment]
What I've done is have the URL goto the actual loveless view that I want them to start on.
